e2e-polybft/e2e/helpers_test.go

```diff
@@ -348,3 +348,20 @@ func getChildToken(t *testing.T, predicateABI *abi.ABI, predicateAddr types.Addr
 
 	return types.StringToAddress(childTokenRaw)
 }
+
+func getLastExitEventID(t *testing.T, relayer txrelayer.TxRelayer) uint64 {
+	t.Helper()
+
+	exitEventsCounterFn := contractsapi.L2StateSender.Abi.Methods["counter"]
+
+	input, err := exitEventsCounterFn.Encode([]interface{}{})
+	require.NoError(t, err)
+
+	initialExitEventIDRaw, err := relayer.Call(ethgo.ZeroAddress, ethgo.Address(contracts.L2StateSenderContract), input)
```

Comment:
Can we change name of these variables to be `lastExitEventIDRaw` and `lastExitEventID`? Later on we can have
`initialExitEventID := getLastExitEventID(t, childchainTxRelayer) + 1`. Its confusing a little bit

Reply:
Done